### PR TITLE
Gatling Tenant Demo [IA-236]

### PR DESCRIPTION
**Pull request by AlexWeinstein92**
A bunch of extra stuff in here
- script for tagging & pushing all images to dockerhub
- script for transferring all images to microk8s `/temp` directory (as a workaround for those without access to dockerhub)
- script for cleaning relevant images in locally microk8s registry (ran out of space at one point)

In addition to the main story - moving Tenant creation & activation to gatling service
Screenshot 2023-07-27 at 10 30 21 AM

### CODE REVIEW

File: gateway/src/main/scala/com/improving/app/gateway/infrastructure/routes/DemoScenarioGatewayRoutes.scala [hunk 1]
```diff
@@ -51,221 +50,165 @@ import scala.concurrent.Future
...
-                  tenants <- tenantResponses
-                  members <- memberResponses
-                } yield ScenarioStarted(tenants = tenants, organizations = Seq(org), members = members).toProtoString
-              }
-            }
-          }
-        }
-      }
-    }
+  // def demoScenarioRoutes(
```
Comment (dag-yoppworks) [inline gateway/src/main/scala/com/improving/app/gateway/infrastructure/routes/DemoScenarioGatewayRoutes.scala:58]: commented out code

File: project/Helpers.scala [hunk 2]
```diff
@@ -173,17 +173,17 @@ object C {
...
     dockerRepository := sys.props.get("docker.registry"),
     dockerUpdateLatest := true,
     dockerExposedPorts ++= Seq(port),
-    dockerCommands ++= Seq(
-      Cmd("FROM", componentName),
-      Cmd("USER", "root"),
-      ExecCmd("RUN", "/sbin/apk", "add", "--no-cache", "bash"),
-      ExecCmd("RUN", "/sbin/apk", "add", "--no-cache", "openjdk17-jre")
-    )
+    // dockerCommands ++= Seq(
```
Comment (dag-yoppworks) [inline project/Helpers.scala:181]: This looks like it's not needed any more with the new base image

**Review by dag-yoppworks — APPROVED** [review]
No show stoppers. I don't love the fixed ip address coming back or the few places with commented out code but if you need to merge as is to continue work I won't hold you up.

File: event/src/main/resources/application.conf [hunk 3]
```diff
@@ -32,6 +32,6 @@ akka.persistence {
 }
 datastax-java-driver.advanced.reconnect-on-init = true
 datastax-java-driver {
-  basic.contact-points = ["scylla-db:9042"]
+  basic.contact-points = ["10.152.183.5:9042"]
```
[the remark this answers is not in the document]
Comment (AlexWeinstein92) [reply event/src/main/resources/application.conf:35]: Perhaps you didn't notice this change in `scyllaApplyForInternalIP`?

Screenshot 2023-07-27 at 12 21 27 PM

By hardcoding the same IP into scylla's service, we do not have to deal with a randomly allocated IP on different machines

File: gateway/src/main/scala/com/improving/app/gateway/infrastructure/routes/DemoScenarioGatewayRoutes.scala (hunk 1 above)
Comment (AlexWeinstein92) [reply gateway/src/main/scala/com/improving/app/gateway/infrastructure/routes/DemoScenarioGatewayRoutes.scala:58]: The is a TODO on top saying when to take this out

File: project/Helpers.scala (hunk 2 above)
Comment (AlexWeinstein92) [reply project/Helpers.scala:181]: Can take it out with the next PR

**Pull request merged**